tests/test_serving.py

```diff
@@ -199,7 +196,7 @@ def test_chunked_encoding(monkeypatch, dev_server, send_length):
     assert data["files"]["file"] == "this is a file"
     environ = data["environ"]
     assert environ["HTTP_TRANSFER_ENCODING"] == "chunked"
-    assert "HTTP_CONTENT_LENGTH" not in environ
+    assert environ.get("CONTENT_LENGTH") == expect_content_len
```

Comment:
The Content-Length header gets exposed as `environ["CONTENT_LENGTH"]`: https://wsgi.readthedocs.io/en/latest/definitions.html#envvar-CONTENT_LENGTH

Reply:
Huh, I see what you were describing now, it's that the key isn't `HTTP_CONTENT_LENGTH`, not sure how I missed that. This makes sense, we're sending an invalid value and want to make sure it's there but not used.